ft_client/freqtrade_client/ft_rest_client.py

```diff
@@ -484,3 +484,17 @@ def health(self):
         :return: json object
         """
         return self._get("health")
+
+    def list_custom_data(self, trade_id=None, key=None):
```

Comment:
key ain't passed to the request?

would it also make sense to implement the /open/custom-data endpoint?